src/controllers/users.js

```diff
@@ -0,0 +1,40 @@
+import Responses from '../utils/response';
+import hash from '../utils/hash';
+import JWTHelper from '../utils/jwt';
+import db from '../models';
+import logger from '../utils/winston';
+
+/**
+ * Class for users related operations
+ */
+export default class {
+  /**
+   * Creates a new user and generate a token
+   * @param {Object} req request object
+   * @param {Object} res response object
+   * @returns {Object} response
+   */
+  static async createUser(req, res) {
+    const passwordHash = hash.generateSync(req.body.password);
+
+    const userData = {
+      firstName: req.body.firstName,
+      lastName: req.body.lastName,
+      email: req.body.email,
+      password: passwordHash
+    };
+
+    const user = await db.user.create(userData).catch((e) => {
+      logger.error(`Database error: ${e.message} - ${e.stack}`);
+      Responses.handleError(500, 'Something went wrong', res);
```

Comment:
You may need to add `return` below this statement to stop JS engine from executing further when there is an error.